.github/workflows/pr_announcement.yml

```diff
@@ -37,16 +37,29 @@ jobs:
                   addComment(input: {
                           subjectId: ${{ env.PR_ID }},
                           body: """Dear contributor,
-                                  Avocado is currently under sprint #103, which is due to release an LTS (Long Term Stability) release. 
+                                  Avocado is currently at the end of sprint #105, therefore we are in feature freeze state.
                                   Please avoid merging changes that do not fall into these categories:
                                   * Bug fixes
-                                  * Usability Improvements
-                                  * Documentation updates
 
-                                  As for the Avocado utility modules (“avocado.utils”) it is OK to introduce new functionality, 
-                                  but changes to the existing APIs (including interface and behavior) should be avoided. 
-                                  These kind of changes should wait until sprint #104."""}) {
+                                  The feature freeze will be active until the release planned on 05/06/2024."""}) {
 
                                   clientMutationId
                   }
           }'
+          #          gh api graphql -f query='mutation {
```

Comment:
This does the job, but, it's really better to remove the content instead of duplicating it and have it commented.  When the time comes to add the LTS notice again, we can recover from version control history.

Reply:
I thought it would be easier when we need LTS notice again, but I don't have a problem with removing it.